storage/EtcCrypttab.h

```diff
@@ -199,9 +199,12 @@ namespace storage
          * Dump the current contents to the log.
          **/
         void log();
+
+    private:
+
+	static const int DEFAULT_PERMISSIONS;
```

Comment:
Having this const in two separate places makes this even harder to read.

This **clearly** belongs very near to the function prototype where it is used. Better yet, don't make this over-complicated; forget that constant and just use the literal value in the function prototype.